### PR TITLE
feat: ✅ add new sections to the landing page

### PR DESCRIPTION
**Pull request by abhagsain**
## Problem / Change Summary 

Added 5 sections along with a footer. 

These sections are 
- Hero Section
- Use Cases 
- Roadmap
- Team Shardeum
- Join Community
- Footer

> What's left?
- In the **Hero Section** we still need to add a video.
- Need to add the left gradient background to the **Roadmap Section**.
- In the **Use Case** section there are two icons I couldn't export correctly. I will sync up with the design team to get the correct icons.

## Screenshot

image

### CODE REVIEW

File: layout/sections/Icons.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
+export const IconSeeMore = () => (
```
Comment (dewanshparashar) [inline layout/sections/Icons.tsx:1]: We should refactor these icons in a common folder, which can be accessed globally.

**Review by dewanshparashar — CHANGES_REQUESTED** [review]
1 FYI comment. And 1 requested change.

**Comment by abhagsain** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> 
> Name	Status	Preview	Updated
> **shardeum-website**	✅ Ready ([Inspect](https://vercel.com/shardeum/shardeum-website/B1bs9H1oqRC1gXRNuBzrSbyvbFkc))	[Visit Preview](https://shardeum-website-git-landing-page-sections-shardeum.vercel.app)	Apr 16, 2022 at 5:16PM (UTC)

image

File: layout/sections/Icons.tsx (hunk 1 above)
Comment (abhagsain) [reply layout/sections/Icons.tsx]: ✅ Done

File: layout/sections/Team.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                  height="300px"
+                />
+              </Box>
+              <VStack alignItems="start">
+                <Text fontSize="xl" color="brand.black" fontWeight="medium">
+                  The Community
+                </Text>
+                <Flex direction="column">
+                  <Text color="brand.grey-70">
+                    Community is the reason how and why Shardeum will remain
```
[the remark this answers is not in the document]
Comment (abhagsain) [reply layout/sections/Team.tsx:75]: 🤝

**Pull request merged**